commodore/compile.py

```diff
@@ -25,57 +26,57 @@
     ApiError,
     clean_working_tree,
     kapitan_compile,
-    lieutenant_query,
 )
 from .postprocess import postprocess_components
 from .refs import update_refs
+from .git import HEAD
 
 
 TARGET = "cluster"
 
 
-def _fetch_global_config(cfg, cluster):
```

Comment:
`fetch_global_config` and `fetch_customer_config` are not part of the public API of `commodore.compile`. They can be left as `_fetch_X_config`.

If they need to be part of the public API for some reason, they should be moved out of the module `commodore.compile` to somewhere where they make sense as part of the API (e.g. `commodore.cluster`).